dataplane/saiserver/acl.go

```diff
@@ -120,6 +121,12 @@ func (a *acl) CreateAclTableGroupMember(_ context.Context, req *saipb.CreateAclT
 // CreateAclTable is noop as the table is already created in the group.
 func (a *acl) CreateAclTable(context.Context, *saipb.CreateAclTableRequest) (*saipb.CreateAclTableResponse, error) {
 	id := a.mgr.NextID()
+
+	a.mgr.StoreAttributes(id, &saipb.AclTableAttribute{
+		AvailableAclEntry:   proto.Uint32(10000),
```

Comment:
These numbers are also used in switch.go and switch_test.go. Should we make these numbers const?

Reply:
they could be different, i just picked the same numbers since they are meaningless, there is no actual limit to acl entries